skia/RSkComponentRNSVGSvgView.cpp

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 1994-2023 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+#include "experimental/svg/model/SkSVGRenderContext.h"
+
+#include "RSkComponentRNSVGGroup.h"
+#include "RSkComponentRNSVGSvgView.h"
+
+namespace facebook {
+namespace react {
+
+RSkComponentRNSVGSvgView::RSkComponentRNSVGSvgView(const ShadowView &shadowView)
+    : INHERITED(shadowView,LAYER_TYPE_PICTURE,SkSVGTag::kSvg) {}
+
+void RSkComponentRNSVGSvgView::OnPaint(SkCanvas *canvas) {
+  SkSVGLengthContext       lctx(SkSize::Make(0, 0));
+  SkSVGPresentationContext pctx;
+
+#ifdef ENABLE_SVG_CHILD_HIERARCHY_DEBUG
+  printChildList();
+#endif
+
+  //1. Set Clip to SVG ontainer size, to restrict child draw within SVG container
```

Comment:
Clipping shouldn't be done  based on maskToBounds property ??

Reply:
Here clipping done to restrict all it child contain with in the SVG  View [Parent SVg container]